cluster/common.sh

```diff
@@ -211,7 +217,7 @@ function get-kubeconfig-user-basicauth() {
 #   KUBE_USER
 #   KUBE_PASSWORD
 function gen-kube-basicauth() {
-    KUBE_USER=admin
+    KUBE_USER=$(admin)
```

Comment:
You're changing from a string "admin" to the results of a program called `admin`? This doesn't seem right, I think it was fine before

Reply:
if not do that,  it will report a error:
In ./cluster/common.sh line 220:
    KUBE_USER=admin
    ^-------^ SC2209: Use var=$(command) to assign output (or quote to assign string).